packages/components/src/SearchParamsSummary/SearchParamsSummary.js

```diff
@@ -46,9 +47,13 @@ export default function SearchParamsSummary({
     <View style={styles.container}>
       <View style={styles.headerLeftContainer}>
         <View style={styles.citiesContainer}>
-          <Text style={styles.departureCity}>{departure?.city || ''}</Text>
+          <Text weight="bold" style={styles.city}>
+            {departure?.city || ''}
```

Comment:
Do we need this fallback?

Reply:
Not sure, this was already in that component. I can try and remove it.

Reply:
Removed all of the fallbacks in that file.